behavior/conditional/dad.go

```diff
@@ -11,6 +12,10 @@ type DadConditional struct {
 
 func (dc *DadConditional) Handle(message string) (string, error) {
 	words := strings.Fields(message)
+	if len(words) > 20 {
```

Comment:
I try to use 'errors' for issues internal to the bot as opposed to user input things - think 500 vs 400 errors.

That being said - what do you think of instead adding a `len(words) < 20` check to the if-statement below this?

Reply:
> I try to use 'errors' for issues internal to the bot as opposed to user input things - think 500 vs 400 errors.

I wasn't entirely sure what the API was, this function didn't return any error yet, so i thought i'd just use it.  I "expected" that returning an error here would yield a private message to the person trying to send the message. like when using a bot on a chat forum and the input is incorrect, you get a message returning an error to the user with 'why' it didn't work/do what they asked it to.

i can change the check to len < 20, sure. updated per that input

Reply:
Definitely agree - this is something I have to clarify in the readme for certain as more people begin to help contribute.